### DISCUSSION TITLE
Separate current articles and archived articles

### DISCUSSION THREAD

**Discussion by jromal** [Lednerb/bilberry-hugo-theme#473] [Q&A] [answered]
Hi all.

I recently discovered this nice Hugo Theme. I like the aspect and simplicity of the overview.
I use it only for one category: "articles". Initially I did planned to show only the last 20 articles in 2 pages of 10.
Then I decided to activate the archive functionality.

I would like that on the main pages only the first 20 articles are shown. And on the archived the rest. 
The articles are in different folders. But I do not get any way to separate them based on the folders where they are.

I had a look to the partials. But being a novice in the area, I could not find a place that would allow me to do this.
Is what I am asking possible with the standards setting of the template? Or changing some partials files? Or with any other workaround?

Many thanks.

**Comment by igor-baiborodine** [comment, marked as answer]
@jromal, the changes you're asking for cannot be implemented using standard configuration properties in the `config.toml` file. Instead, you should modify the `layouts/archive/single.html` and `layouts/index.html` templates and copy them to your repository. Please look at this [PR](https://github.com/igor-baiborodine/bilberry-hugo-theme-sandbox/pull/29/files) request I created to implement the changes you asked for in the Bilberry Sandbox. 

These changes can be tested here: https://deploy-preview-29--bilberry-sandbox.netlify.app/

**Reply by jromal** [reply]
Many thanks. I did missed to check the "index.html". 
I managed to show only the first 20 articles. I created a new parameter on the config file to make it more flexible.

I did not managed to make it work that the archive does not show the 20 shown on the archive. Somehow the changes did not have any effect. But this is not really something it bothers me. I am happy with only showing 20 articles on two pages overview.

**Reply by igor-baiborodine** [reply]
These lines will get you all articles except for the first 20: 
- https://github.com/igor-baiborodine/bilberry-hugo-theme-sandbox/blob/d00660355b7246281d9212c8afc34a8a3089cd81/layouts/archive/single.html#L14
- https://github.com/igor-baiborodine/bilberry-hugo-theme-sandbox/blob/d00660355b7246281d9212c8afc34a8a3089cd81/layouts/archive/single.html#L15

**Reply by jromal** [reply]
Thanks. I missed the changes because L13 was also missing. And I forgot to change L15.
Now it works like a charm.
